conf/inventory/ibm-rhel-9.5-server-bx2-2x8.yaml

```diff
@@ -0,0 +1,44 @@
+---
+version_id: 9.5
+id: rhel
+instance:
+  create:
+    image-name: rhel-95-server-amd64-kvm
+    network_name: ci-sn-01
+    private_key: ceph-qe-sa-svc
+    group_access: sec-grp-ci-vpc-01
+    profile: bx2-4x16
```

Comment:
The expected profile value is likely bx2-2x8. Kindly verify.

Reply:
fixed